### PR TITLE
Integration get started page

### PR DESCRIPTION
**Pull request by zhyatt**
Review Integration Guides index page

```
docker pull ghcr.io/nanocurrency/nano-docs:pr-566
docker run --rm -it -p 8000:8000 ghcr.io/nanocurrency/nano-docs:pr-566
```

### CODE REVIEW

**Comment by filipesmedeiros** [comment]
I'll take a look as soon as I have a free slot!

**Comment by qwahzi** [comment]
Some initial thoughts. Will edit with more notes:

**"What is Nano? Overview" page**

- ~~"Pending", "Proof-of-Work", "Representative", "Contributing", "running a node", "review integration options", "RPC", and "CLI" links didn't work because of missing trailing slash~~ [Confirmed Docker artifact after testing on live docs.nano.org]

- Living Whitepaper link is broken because it's looking for /living-whitepaper/index.md vs /living-whitepaper/ [Confirmed issue, not just Docker artifact]

- Pending should be changed to "Receivable / Ready to Receive" per https://github.com/nanocurrency/nano-node/issues/3411

- The receiver can be offline and safely leave the funds in this state until they are ready to publish a matching block receiving the funds to their account, **though best practice is to publish receive blocks as soon as possible so that the balance participates in consensus, and so that optional pruning is more effective(?)** [Possible suggestion, but might be overkill for an overview page]

**"Glossary" page**
- Pending should point to "Receivable / Ready to Receive", per https://github.com/nanocurrency/nano-node/issues/3411

**Comment by zhyatt** [comment]
Awesome, thanks for the initial feedback @qwahzi ⭐ Will review and reply once you have a chance to provide additional notes.

**Comment by zhyatt** [comment]
Thanks for helping @filipesmedeiros !

**Comment by satoshizzle** [comment]
I might be wrong but In https://github.com/nanocurrency/nano-docs/blob/master/docs/what-is-nano/overview.md

"Once a block sending funds is confirmed by the network, the transaction goes into a **pending** state and cannot be reversed"

Shouldn't that be replaced with "ready to receive" state? 

https://www.reddit.com/r/nanocurrency/comments/ov3fro/term_pending_to_be_replaced_with_ready_to_receive/

**Comment by zhyatt** [comment]
@satoshizzle Thanks, I will include this when I review all feedback shortly. We have a separate issue related to this to update throughout the docs soon: #554

**Comment by mistakia** [comment]
The `Integration Guides` link on the homepage should probably link to the index page instead of `The Basics` page.

**Comment by zhyatt** [comment]
@qwahzi I think I covered your items, some notes:
- Unit updates merged from #466 should resolve both those items
- Pending/receivable changes have been noted for addressing as part of #554 
- Best practice for receiving transactions on What is Nano page did feel out of place, so didn't include

@satoshizzle @mistakia Your comments should also be resolved now.

File: docs/snippets/warning-security-vulnerability-reporting.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+!!! warning "Submit vulnerabilities privately"
+	**Do NOT discuss potential security vulnerabilities on the issue tracker, public forums or open discussion channels. Submit sensitive issues privately to the Nano Foundation for review.**
```
Comment (filipesmedeiros) [inline docs/snippets/warning-security-vulnerability-reporting.md:2]: Maybe add here a way to contact them? Email or a link?

File: docs/integration-guides/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+If going this route, we encourage use of the existing test network for initial integrations. Head over to the [Running a node](../running-a-node/overview.md) guide and make your way through the overview and security pages before stepping through the node setup guide. Make sure you have "Test network" selected in all the example commands.
+
+!!! info "Production integration node should be non-voting"
+	When moving to production with a node on the main network, we recommend running dedicated, non-voting node for your integration. If you are interested in running a representative node to help further decentralize the network consensus, please setup a separate node for this purpose to ensure both operate as effectively as possible. See the [Voting as a Representative guide](../running-a-node/voting-as-a-representative.md) for further details.
+
+### Public APIs
+
+Access to public APIs for the main network is also available via third-party vendors and community members. These are only available for the main network and caution should be taken when evaluating SLAs and uptime in general.
```
Comment (filipesmedeiros) [inline docs/integration-guides/index.md:58]: Maybe add that this is the preferred way of quickly testing something out or build a PoC?

Comment (zhyatt) [reply docs/integration-guides/index.md]: Added a note about this, thanks!

**Comment by zhyatt** [comment]
I am looking to merge this soon so if there are any remaining points of feedback, please drop them in. Otherwise, marking "Approved" on the review is much appreciated!

**Comment by otonashixav** [comment]
I might suggest using strictly IPv4 for the docker setup, possibly moving IPv6 configuration to an additional setup step. I quickly ran through the setup process and ran into issues querying the RPC since IPv6 is not enabled by default (I recall some users on discord having this issue) and not supported at all on windows hosts (me while testing). Removing `[::1]` from the docker run command allowed the rpc to be queried.

**Comment by otonashixav** [comment]
1. I might suggest using strictly IPv4 for the docker setup, possibly moving IPv6 configuration to an additional setup step. I quickly ran through the setup process and ran into issues querying the RPC since IPv6 is not enabled by default (I recall some users on discord having this issue) and not supported at all on windows hosts (me while testing). Using `[::1]` in the curl command did not work, whereas specifying `-p 127.0.0.1:17076:17076` allowed the RPC to be queried. 
2. Since docker is the suggested method of running a node, it would be nice if commands such as `nano_node --generate_config node` in the Node Configuration section showed how to run them with docker i.e. `docker exec container_name nano_node --generate_config node`.

**Comment by zhyatt** [comment]
> 1. I might suggest using strictly IPv4 for the docker setup, possibly moving IPv6 configuration to an additional setup step. I quickly ran through the setup process and ran into issues querying the RPC since IPv6 is not enabled by default (I recall some users on discord having this issue) and not supported at all on windows hosts (me while testing). Using `[::1]` in the curl command did not work, whereas specifying `-p 127.0.0.1:17076:17076` allowed the RPC to be queried.

Added to new issue for consideration https://github.com/nanocurrency/nano-docs/issues/572

> 2. Since docker is the suggested method of running a node, it would be nice if commands such as `nano_node --generate_config node` in the Node Configuration section showed how to run them with docker i.e. `docker exec container_name nano_node --generate_config node`.

Added to new issue as well https://github.com/nanocurrency/nano-docs/issues/574

Thanks @otonashixav

**Pull request merged**